peru/runtime.py

```diff
@@ -139,6 +144,67 @@ def print_overrides(self):
             self.display.print('  {}: {}'.format(
                 name, self.get_override(name)))
 
+    def watch_overrides(self):
+        class WatchHandler(LoggingEventHandler):
+            logging.basicConfig(level=logging.INFO,
+                            format='%(asctime)s - %(message)s',
+                            datefmt='%Y-%m-%d %H:%M:%S')
+
+            def on_moved(self, event):
+                super(LoggingEventHandler, self).on_moved(event)
+                what = 'directory' if event.is_directory else 'file'
+                logging.info('moved %s: from %s to %s', what, event.src_path,
+                             event.dest_path)
+                logging.info('running peru sync --force')
```

Comment:
The `sync` and `reup` commands use a fancy display to draw to the terminal and clean things up as jobs finish. This conflicts with regular printing. The display object (usually `runtime.display`) has a `print` method that lets us do regular printing in a way that doesn't screw up the redrawing.